lib/ProductOpener/Config2_sample.pm

```diff
@@ -76,4 +78,12 @@ $crowdin_project_key = '';
 # enable an in-site robotoff-asker in the product page
 $robotoff_url = '';
 
+%server_options = (
```

Comment:
Yes, same for Config.pm : overtime, I'd like to move all the non-required fields to the %options and %server_options hashes. The goal is to avoid having to change all Config files whenever we want to add a new option.